### ISSUE TITLE
upload_set_form_field  params doesn`t work

### ISSUE THREAD

**Issue by fallenner** [vkholodkov/nginx-upload-module#89]
the nginx.conf 
location /upload {
            # Pass altered request body to this location
            upload_pass   @test;

            # Store files to this directory
            # The directory is hashed, subdirectories 0 1 2 3 4 5 6 7 8 9 should exist
            upload_store /tmp/nginx_upload;
            
            # Allow uploaded files to be read only by user
            upload_store_access user:r;
		upload_pass_args off;
            # Set specified fields in request body
            upload_set_form_field "${upload_field_name}_name" $upload_file_name; 
           upload_set_form_field "${upload_field_name}_content_type" $upload_content_type; 
         upload_set_form_field "${upload_field_name}_path" $upload_tmp_path;

            # Inform backend about hash and size of a file
            upload_aggregate_form_field "${upload_field_name}_md5" $upload_file_md5;
            upload_aggregate_form_field "${upload_field_name}_size" $upload_file_size;

            upload_pass_form_field "^submit$|^description$";
         }

        # Pass altered request body to a backend
        location @test {
            proxy_pass   http://192.168.0.103:3000;
        }

 the request body is empty  http://192.168.0.103:3000;   nginx version  1.8.1

**Comment by lapwat** [comment]
@fallenner Did you find out how to do it ?

**Comment by fallenner** [comment]
@lapwat  yes. I hava solved. it's because miss openssl-dev lib.  I pack it.  you can use it easier on my repository.   [https://github.com/fallenner/nginx-file-server](nginx-file-server)

**Comment by lapwat** [comment]
Cheers!

**Comment by vfa-anhnvt** [comment]
> Cheers!
> 
> Edit: Did not work even with openssl...

Hi apwat , i meet same proplem with nodejs. I install openssl-dev lib, too. But can't  solved proplem.   😭. Please help me..

**Comment by lapwat** [comment]
@vfa-anhnvt Could not make it work sorry, I got rid of nginx.
